foundation/foundation-configuration-store/README.md

```diff
@@ -7,21 +7,41 @@ please refer to the [aiSSEMBLE&trade; Github Pages](https://boozallen.github.io/
 
 ## Developer Guidance
 
-### Deserializing YAML Content into Java Objects with snakeyaml
-The `ConfigLoader` leverages [snakeyaml](https://bitbucket.org/snakeyaml/snakeyaml/src/master/) to parse `YAML` files define configurations and associated metadata. This module contains two classes, `YamlProperty` and `YamlConfig`, which are the objects that house the deserialized information from the configuration `YAML` files. An example of properly formatted configurations files:
+### Deserializing Properties Content into Java Objects with Krausening
+The `ConfigLoader` leverages [Krausening](https://github.com/TechnologyBrewery/krausening/tree/dev/krausening) to parse properties files define configurations and associated metadata, which enables the configuration value encryption for security reasons. 
+When defining a property, the property name should be the group name and property content should be in key=value format. An example of properly formatted configurations files:
```

Comment:
A: This line seems incorrect but I'm not sure what we're trying to say. Specifically "_the property name should be the group name_" sounds off. I don't know what that means.